Rakefile

```diff
@@ -11,6 +11,7 @@ Hoe.spec 'pmdtester' do
 
   self.author  = 'Binguo Bao'
   self.email   = 'djydewang@gmail.com'
+  self.clean_globs = %w[target/reports/**/* target/test**/*]
```

Comment:
You can simply clean the whole target directory, no need to be specific here :smile:

Reply:
Maybe we shouldn't clone standard projects to `target/repositories` again and again.

Reply:
OK, I understand. Hm... this sound like a future improvement - we should make the path, to which we clone the projects, configurable. Currently it is [fixed](https://github.com/pmd/pmd-regression-tester/blob/4b6b53953048d6578b59e6cab7b01f6826275841/lib/pmdtester/builders/pmd_report_builder.rb#L17).

So, not cleaning the whole `target` directory means, we get a slight performance improvements for the second integration test...